source/time.tex

```diff
@@ -1133,9 +1133,9 @@
 template<class Rep>
   struct duration_values {
   public:
-    static constexpr Rep zero();
-    static constexpr Rep min();
-    static constexpr Rep max();
+    static constexpr Rep zero() noexcept;;
```

Comment:
The extra semicolon should not be there?

(I have no experience with the C++ standardization process, so excuse me if this is the wrong way to point out a typo.)

Reply:
Thanks for catching this!!